src/cclib/parser/daltonparser.py

```diff
@@ -0,0 +1,277 @@
+        #
+        # the line:
+        # *** SCF orbital energy analysis ***
+        # triggers the analysis
+        #
+        if "*** SCF orbital energy analysis ***" in line:
+
+            # to get ALL orbital energies, the .PRINTLEVELS keyword needs
+            # to be at least 0,10 (up from 0,5). I know, obvious, right?
+            # this, however, will conflict with the scfvalues output that
+            # changes into some weird form of DIIS debug output.
+
+            mosyms = []
+            moenergies = []
+
+            line = next(inputfile)
+            # if the second line has "Only the five" present in it, then we have a reduced output in
+            # the number of orbital energies.
+            line = next(inputfile)  # has number of electrons if we need it
+            if "Only the five" in line:
+                line = next(inputfile)
+                line = next(inputfile)
+            
+            line = next(inputfile)  # has # of orbital symmetries
+            temp = line.split()
+            nsym = len(temp[3:])
+
+            line = next(inputfile)
+            line = next(inputfile)
+            line = next(inputfile)
+
+            # now parse nsym symmetries
+            while nsym > 0:
+
+                line = next(inputfile)
+                temp = line.split()
+
+                # there seems to be an errornous printout of a newline character in DALTON
+                # if the number of orbital energies printed for a given symmetry is 5
+                if len(temp) == 0:
+                    continue
+
+                # this first line has the orbital symmetry information
+                sym = self.normalisesym(temp[1])
+                temp = [float(t) for t in temp[2:]]
+                moenergies.extend(temp)
+                mosyms.extend(len(temp)*[sym])
+                while len(temp) > 0:
+                    line = next(inputfile)
+                    temp = [float(col) for col in line.split()]
+
+                    if len(temp) == 0:
+                        continue
+
+                    moenergies.extend(temp)
+                    mosyms.extend(len(temp)*[sym])
+
+                nsym -= 1
+
+            # now sort the data about energies and symmetries. see the following post for the magic
+            # http://stackoverflow.com/questions/19339/a-transpose-unzip-function-in-python-inverse-of-zip
+            sdata = sorted(zip(moenergies, mosyms), key=lambda x: x[0])
+            moenergies, mosyms = zip(*sdata)
+
+            self.moenergies = [[]]
+            self.moenergies[0] = moenergies
+            self.mosyms = [[]]
+            self.mosyms[0] = mosyms
+
+            if not hasattr(self, "nmo"):
+                self.nmo = self.nbasis
+                if len(self.moenergies[0]) != self.nmo:
+                    self.set_attribute('nmo', len(self.moenergies[0]))
+
+            self.mocoeffs = [numpy.zeros((self.nmo, self.nbasis), "d")]
```

Comment:
If this is going to be merged in as-is, we probably want to implement this right away, otherwise it should get removed.